internal/generator/vector/output/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,133 @@
+package elasticsearch
+
+import (
+	"strings"
+
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	. "github.com/openshift/cluster-logging-operator/internal/generator"
+	"github.com/openshift/cluster-logging-operator/internal/generator/fluentd/helpers"
```

Comment:
need this helper along with vector helper?

Reply:
Moved method to vector helpers now.